core/amber/src/main/scala/edu/uci/ics/texera/workflow/common/operators/OperatorDescriptor.scala

```diff
@@ -77,7 +77,7 @@ import edu.uci.ics.texera.workflow.operators.sink.managed.ProgressiveSinkOpDesc
     new Type(value = classOf[WordCloudOpDesc], name = "WordCloud"),
     new Type(value = classOf[HtmlVizOpDesc], name = "HTMLVisualizer"),
     new Type(value = classOf[ScatterplotOpDesc], name = "Scatterplot"),
-    new Type(value = classOf[PythonUDFOpDesc], name = "PythonUDF"),
+    new Type(value = classOf[PythonUDFOpDesc], name = "PythonUDFV1"),
```

Comment:
This will break the existing workflows using V1 and break the DB, I thought this is not what you wanted. This is the `operatorType` not the title, if you want to just change the title go to `PythonUDFOpDesc.operatorInfo`

Reply:
changed to `PythonUDFOpDesc.operatorInfo`